R/utils.R

```diff
@@ -178,3 +180,33 @@ read_tabular_file <- function(path_to_file,
 
   return(file_reader(path_to_file, ...))
 }
+
+#' @rdname read_tabular_file
+#' @param table Table to save, passed to the writer function.
+#' @return Nothing, saving the table as a side
+#' effect.
+#' @export
+write_tabular_file <- function(table,
+                               path_to_file,
+                               file_format = NULL,
+                               ...) {
+  if (is.null(file_format)) {
+    file_format <- fs::path_ext(path_to_file)
+  }
```

Comment:
I think better to always do `file_format <- fs::path_ext(path_to_file)` and not have `file_format` as a function arg. No need to make things overly-complicated.